.travis.yml

```diff
@@ -15,23 +15,23 @@
 
 language: java
 sudo: false
-
+cache:
+  directories:
+    - $HOME/.m2
 matrix:
   include:
     - os: linux-ppc64le
       jdk: openjdk8
-
-jdk:
-  - oraclejdk11
-  - openjdk8
-  - openjdk11
-  - openjdk12
-  - openjdk13
-  - openjdk14  
-  - openjdk-ea
-
+    - jdk: openjdk8
+    - jdk: openjdk11
+    - jdk: openjdk12
```

Comment:
Can you drop:
```
openjdk12
openjdk13
oraclejdk11
```

Now Java 14 is out 12 and 13 are redundant. Testing on oraclejdk11 and openjdk11 also seems redundant.

Reply:
> Now Java 14 is out 12 and 13 are redundant. Testing on oraclejdk11 and openjdk11 also seems redundant.

we can drop 12 but 13 but I think we'd better remain 11, as 11 is LTS.

Reply:
My point was that testing on **both** oraclejdk11 and openjdk11 seems redundant. I would also drop oraclejdk11 and just leave openjdk11.

Reply:
> My point was that testing on **both** oraclejdk11 and openjdk11 seems redundant. I would also drop oraclejdk11 and just leave openjdk11.

@aherbert oh, got it:)